hands-on-exercise/3-cosmjs-adv/2-cosmjs-messages.md

```diff
@@ -15,7 +15,7 @@ Make sure you have all you need before proceeding:
 
 * You understand the concepts of [CosmJS](/tutorials/7-cosmjs/1-cosmjs-intro.md).
 * You have generated the necessary TypeScript types in [the previous tutorial](./1-cosmjs-objects.md).
-* You have the finished the checkers blockchain exercise. If not, you can follow that tutorial [here](/hands-on-exercise/1-ignite-cli/index.md) or just clone and checkout the [relevant branch](https://github.com/cosmos/b9-checkers-academy-draft/tree/wager-denomination) that contains the final version.
+* You have the finished checkers blockchain exercise. If not, you can follow that tutorial [here](/hands-on-exercise/1-ignite-cli/index.md) or just clone and checkout the [relevant branch](https://github.com/cosmos/b9-checkers-academy-draft/tree/v1-wager-denomination) that contains the final version.
```

Comment:
Wait no, it should not be v1

Reply:
Fixed here https://github.com/cosmos/sdk-tutorials/pull/1339/files#diff-78cf44d53f0679346a9032b2596d806dcf762eba69dc4bfc7d6ff17265d6c71b